lib/slack_commands/status_command.rb

```diff
@@ -18,10 +18,15 @@ def run
           "*free*"
         end
 
-      inactive_time = heroku.last_active_at(app)
-      active_message = inactive_time ? time_ago_in_words(inactive_time) : "a while"
+      message = "#{app}: #{claim_message}"
+
+      if HEROKU_APPS.include? app
+        inactive_time = heroku.last_active_at(app)
+        active_message = inactive_time ? time_ago_in_words(inactive_time) : "a while"
+
+        message = "#{message} (last active #{active_message} ago)"
```

Comment:
I'd actually just remove this whole section– the "last active" bit isn't very accurate and we've talked about killing it in the past anyway.